Add Helm chart installation support (#32)

rajanarahul93 opened the pull request:
Implements Helm chart installation functionality as requested in #32.

Features:
- Install Helm charts with custom values
- Uninstall Helm releases
- Upgrade existing releases
- Support for namespaces
- Support for version specification
- Support for custom repositories

Added test file `helm.test.ts` for testing Helm operations.

Flux159 commented on src/helm.test.ts:
```diff
@@ -0,0 +1,58 @@
+    }
+  });
+
+  test("install helm chart", async () => {
+    // TODO: Implement test for installing a helm chart
+  });
+
+  test("uninstall helm chart", async () => {
+    // TODO: Implement test for uninstalling a helm chart
+  });
+
+  test("update helm chart values", async () => {
+    // TODO: Implement test for updating helm chart values
+  });
```
Can we add some simple tests here before merging? Maybe use the bitnami nginx helm chart since it should be relatively lightweight - https://github.com/bitnami/charts/tree/main/bitnami/nginx

Flux159 reviewed (commented):
Code LGTM, just want to make sure we have tests before merging - so could you update that & I'll approve then? Thanks!

rr-paras-patel commented:
Please update README.md as well and be specific about helm v3 support without tiller

rajanarahul93 commented:
I've made the requested changes:
1)Added comprehensive tests using the Bitnami nginx helm chart:
Test for installing a chart with custom values
Test for upgrading a chart with new values
Test for uninstalling a chart
Added proper cleanup in afterEach hook
Added appropriate timeouts for long-running operations
2)Updated the README.md to:
Specify Helm v3 support (no Tiller required)
Add Helm v3 as a prerequisite
Update the features list to show Helm support as completed
Add detailed Helm features
Update project structure to include helm.test.ts

Once check and Merge

Flux159 commented:
@rajanarahul93 - I updated the PR with a commit to fix the tests in https://github.com/Flux159/mcp-server-kubernetes/pull/36 - see the changes in https://github.com/Flux159/mcp-server-kubernetes/pull/36/commits/cb6a4f76de5855285a62a639ea33665d9719d9fa

Thanks! Will make release 0.1.5 with these changes in a bit.
